### ISSUE TITLE
Cronofy generates massive log spam via Hashie

### ISSUE THREAD

**Issue by rkischuk** [cronofy/cronofy-ruby#51]
Our logs are absolutely overwhelmed with the following log message:
```
W, [2017-10-13T11:29:09.816974 #6930]  WARN -- : You are setting a key that conflicts with a built-in method Cronofy::PagedEventsResult#update defined at /home/deploy/Acadia/shared/bundle/ruby/2.3.0/gems/hashie-3.5.6/lib/hashie/mash.rb:209. This can cause unexpected behavior when accessing the key via as a property. You can still access the key via the #[] method.
```

Like 3 million times in 24 hours.

It makes the logs nearly useless and fills up the server disk space until the server crashes.

While this is due to a silly decision by Hashie, changes are needed in Cronofy to suppress it.

Cronofy needs to subclass `Hashie::Mash` and `disable_warnings` - https://github.com/intridea/hashie/issues/394

**Comment by gshutler** [comment]
You could try referencing the branch from #52 

We'll get a release out some point today.

**Comment by gshutler** [comment]
This was fixed in [0.24.1](https://rubygems.org/gems/cronofy/versions/0.24.1).

**Comment by rkischuk** [comment]
Amazing turnaround time. Thanks, @gshutler!
